src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -355,6 +380,16 @@ export const QueryResultPane = () => {
                         </Table>
                     </div>
                 )}
+                {metadata.tabStates!.resultPaneTab ===
```

Comment:
why a null assertion here?

Reply:
I was mostly just following what they did for the other tabs, but if i'm understanding correctly, it's for the initial load in when the state might not be defined yet.

Reply:
The issue where state may not be defined yet _should_ be fixed now with [this](https://github.com/microsoft/vscode-mssql/pull/18147/files#diff-c03515c44e865bf8d0d3ed19221a87854595b4505ce58559e3e5f0d9f13d8f59) change, but no need to go through and remove all the checks in this PR.

Unresolving this comment for vis, but you can just mark it as resolved again.